.github/workflows/release-quil-py.yml

```diff
@@ -0,0 +1,109 @@
+name: Release quil-py
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    description: "Manually publish release"
+
+
+jobs:
+  is-python-release:
+    if: ${{ github.event_name == 'workflow_dispatch' || startsWith(github.event.release.tag_name, 'quil-py/v') }}
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Release tag starts with quil-py/v, proceeding with release"
+
+  macos:
+    runs-on: macos-11
```

Comment:
Any specific reason for `macos-11` over `macos-latest`, considering the use of `ubuntu-latest`?

Reply:
Pinned them to `macos-12` and `ubuntu-22.04`